openfecli/tests/commands/test_gather.py

```diff
@@ -275,7 +275,7 @@ def test_cmet_failed_edge(self, cmet_result_dir, report, file_regression):
 
     @pytest.mark.parametrize("allow_partial", [True, False])
     def test_cmet_too_few_edges_error(self, cmet_result_dir, allow_partial):
-        results = [str(cmet_result_dir / f"result_{i}_failed_edge") for i in range(3)]
+        results = [str(cmet_result_dir / f"results_{i}_failed_edge") for i in range(3)]
```

Comment:
this is a fix, not a behavior change from this PR. 
before, it wasn't detecting any edges because of a typo in the file path. 

I think the fact that this PR caught this proves its value!